zerver/tests/test_auth_backends.py

```diff
@@ -199,11 +199,7 @@ def test_email_auth_backend(self) -> None:
                             good_kwargs=dict(password=password,
                                              username=username,
                                              realm=get_realm('zulip'),
-                                             return_data=dict()),
-                            bad_kwargs=dict(password=password,
-                                            username=username,
-                                            realm=None,
-                                            return_data=dict()))
+                                             return_data=dict()))
```

Comment:
We be replacing `realm=None` with `realm=get_realm("zephyr")` for these tests.

Reply:
Replaced.